frontend/src/metabase/actions/utils.ts

```diff
@@ -254,19 +252,6 @@ export const getFormTitle = (action: WritebackAction): string => {
   return action.visualization_settings?.name || action.name || t`Action form`;
 };
 
-export function setNumericValues(
-  params: ParametersForActionExecution,
-  fieldSettings: FieldSettingsMap,
-) {
-  Object.entries(params).forEach(([key, value]) => {
-    if (fieldSettings[key]?.fieldType === "number" && !isEmpty(value)) {
-      params[key] = Number(value) ?? null;
-    }
-  });
-
-  return params;
-}
-
```

Comment:
Removed in favor of [`formatSubmitValues`](https://github.com/metabase/metabase/blob/f9c9003ff12098b4096dbcac936759d420a85e69/frontend/src/metabase/actions/hooks/use-action-form/utils.ts#L33)